traitsui/qt4/tree_editor.py

```diff
@@ -23,7 +23,8 @@
 
 from pyface.qt import QtCore, QtGui
 
-from pyface.resource_manager import resource_manager
+from pyface.api import ImageResource
+from pyface.ui_traits import convert_image
```

Comment:
I have an `ImportError: No module named ui_traits` here, with `pyface==5.1.0`, which is the latest version apparently.

Reply:
You'll need Pyface master for current TraitsUI code to work.

Reply:
Ok thanks